client/websocket_client.jsx

```diff
@@ -119,6 +126,9 @@ export default class WebSocketClient {
                 this.eventSequence = msg.seq + 1;
                 this.eventCallback(msg);
             }
+
+            this.clearPingPong();
+            this.createPingEvent();
```

Comment:
should we repeat this when we send a message? (line 182, sorry but github doesn't allow me to add the comment there)

Reply:
Good question, I didn't think this before but I would say No, because sending doesn't confirm that socket is open